util/src/main/java/org/datacommons/util/McfParser.java

```diff
@@ -0,0 +1,291 @@
+package org.datacommons.util;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.apache.commons.io.IOUtils;
+import org.datacommons.proto.Mcf;
+import org.datacommons.proto.Mcf.McfGraph;
+import java.io.IOException;
+import java.io.StringReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+// TODO: Implement COMPLEX_VALUE parsing
+// TODO: Implement TEMPLATE_MCF parsing
+public class McfParser {
```

Comment:
Add comments for class and public methods?

Reply:
Done.